### DISCUSSION TITLE
Getting rejected again and again

### DISCUSSION THREAD

**Discussion by NaviTheCoderboi** [community/community#56554] [GitHub Education]
### Hi everyone,

When i apply for GitHub developer pack i get rejected, i got rejected 5 times , i get this response
```
Unfortunately, we weren't able to approve your educational discount request for @NaviTheCoderboi.


What happened?
Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.
```
Even my documents are correct 
I provided picture of my school identity card
I am in grade 9 and 14, years old , please tell me what should I do to fix it 
Thank you

**Comment by samus-aran** [comment]
Hi @NaviTheCoderboi ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same [comment in multiple places](https://github.com/orgs/community/discussions/56181). As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
